src/networking.c

```diff
@@ -2166,7 +2166,9 @@ void readQueryFromClient(connection *conn) {
             return;
         }
     } else if (nread == 0) {
-        serverLog(LL_VERBOSE, "Client closed connection");
+        sds info = catClientInfoString(sdsempty(), c);
```

Comment:
i agree it can be useful for debugging, but considering that this log print is normally filtered (VERBOSE is not the default), and the cost of calling catClientInfoString, i think it may be better to put that block inside an `if` for the log level.

Reply:
@oranagra thanks, already updated.